packages/aa/src/index.js

```diff
@@ -12,10 +12,19 @@ module.exports = {
   createPerformantResolve,
 }
 
+/** @type {Resolver|undefined} */
+let performantResolve
```

Comment:
```suggestion
/** @type {Resolver|undefined} */
let performantResolve
```
I'm guessing this global state acts as a memoization-optimization? Could we avoid making it mutable by instantiating it immediately?
```suggestion
/** @type {Resolver} */
const performantResolve = createPerformantResolve()
```

Reply:
yeah, seems good